src/pages/index.tsx

```diff
@@ -18,10 +22,18 @@ const Home: NextPage = () => {
             }
 
             try {
-                await axios.get(`${process.env.NEXT_PUBLIC_API}/auth/me`, {
+                setIsLoading(true);
+
+                const res = await axios.get(`${process.env.NEXT_PUBLIC_API}/auth/me`, {
```

Comment:
I would have extract all api calls into some http utility file which will expose get, post, patch, delete methods and use that here. Even better to remove the api call directly within the component.
- /src/api/
  - http.js [exposes get, post, patch, delete method]
  - auth.js [exposes auth api call e.g. getLoggedInUser which will manage the api call {api}/auth/me]
  - project.js
  - org.js
Later on, if you want to replace axios with some other library, it will be easier.